openmetadata-ui/src/main/resources/ui/src/styles/app.less

```diff
@@ -161,7 +164,7 @@
   align-items: center;
 }
 .justify-end {
-  justify-content: flex-end;
+  justify-content: flex-end !important;
```

Comment:
why did we need to add important to global style? it can lead to unexpected behaviour!